src/styles/utils/variables.scss

```diff
@@ -0,0 +1,7 @@
+$mainTextColor: #060b35;
+$imageWidth: 160px;
+$imageHeight: 134px;
+$starsAmount: 4;
+$numberOfColums: 4;
```

Comment:
There is a typo in the variable name `$numberOfColums`. It should be `$numberOfColumns` to ensure consistency and avoid errors in the stylesheets.